src/features/game/lib/gameMachine.ts

```diff
@@ -669,10 +796,39 @@ export function startGame(authContext: Options) {
               saveGame(
                 context,
                 event,
-                authContext.farmId as number,
-                authContext.rawToken as string
+                authContext.user.farmId as number,
+                authContext.user.rawToken as string
               ),
+            onDone: [
+              {
+                target: "playing",
+                actions: assign((context: Context, event) =>
+                  handleSuccessfulSave(context, event)
+                ),
+              },
+            ],
+            onError: {
+              target: "error",
+              actions: "assignErrorMessage",
+            },
+          },
+        },
+        guestAutosaving: {
```

Comment:
What are the key differences between `autosaving` and `guestAutosaving`?

Is it possible to have a single `autosaving` and handle the API branching inside of `autosave.ts`?